### ISSUE TITLE
woeusbgui won't run with error in source file

### ISSUE THREAD

**Issue by iKarith** [WoeUSB/WoeUSB-ng#3]
* [X] I've searched the issue tracker and is pretty sure that there's no duplicate issue already filed
* [ ] I've built the latest development snapshot using the instructions in README and verified that the issue can still be reproduced (for bug reports)

## Issue Reproduce Instructions
> 1. Launch WoeUSBGUI by running `woeusbgui`
> 2. Receive the following error immediately:
```
  File "/usr/local/bin/woeusbgui", line 5
SyntaxError: Non-ASCII character '\xc2' in file /usr/local/bin/woeusbgui on line 6, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
```

## Expected Behavior
Expected the GUI to start.

## Current Behavior
GUI does not start.

### WoeUSB Version
```
tjcarter@chiyo:~$ pip3 show woeusb-ng
Name: WoeUSB-ng
Version: 0.2.0
Summary: WoeUSB-ng is a simple tool that enable you to create your own usb stick windows installer from an iso image or a real DVD. This is a rewrite of original WoeUSB. 
Home-page: http://github.com/WoeUSB/WoeUSB-ng
Author: Jakub Szymański
Author-email: jakubmateusz@poczta.onet.pl
License: GPL-3
Location: /usr/local/lib/python3.8/dist-packages
Requires: termcolor, wxPython
Required-by: 
```

### WoeUSB Source
Installed using `pip3 install woeusb-ng`

### Information about the Operating System
```
tjcarter@chiyo:~$ lsb_release -d
Description:    Debian GNU/Linux bullseye/sid
tjcarter@chiyo:~$ python3 --version
Python 3.8.2
tjcarter@chiyo:~$ dpkg -l grub-pc-bin grub-efi-amd64-bin ntfs-3g dosfstools git p7zip-full python3-pip python3-wxgtk4.0
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name               Version           Architecture Description
+++-==================-=================-============-==================================================================
ii  dosfstools         4.1-2             amd64        utilities for making and checking MS-DOS FAT filesystems
ii  git                1:2.25.1-1        amd64        fast, scalable, distributed revision control system
ii  grub-efi-amd64-bin 2.04-5            amd64        GRand Unified Bootloader, version 2 (EFI-AMD64 modules)
ii  grub-pc-bin        2.04-5            amd64        GRand Unified Bootloader, version 2 (PC/BIOS modules)
ii  ntfs-3g            1:2017.3.23AR.3-3 amd64        read/write NTFS driver for FUSE
ii  p7zip-full         16.02+dfsg-7      amd64        7z and 7za file archivers with high compression ratio
ii  python3-pip        18.1-5            all          Python package installer
ii  python3-wxgtk4.0   4.0.7+dfsg-2      amd64        Python 3 interface to the wxWidgets Cross-platform C++ GUI toolkit
```

> Different OS distributions provides different characteristics and may influence the result of running WoeUSB.  Run `lsb_release --description` or `lsb_release -d` in a terminal to acquire this information.

### Information about the Source Media
Windows 10 build 1909, downloaded from Microsoft directly

### Information about the Target Device
SanDisk 16GB

**Comment by WaxyMocha** [comment]
Fixed in 606bb891823fbf6f966008e1c24b513d928cfec1, can you confirm?

**Comment by iKarith** [comment]
I can, thanks. Testing a write from the GUI as I type this.

**Comment by iKarith** [comment]
That resulted in … um, this:

![Screenshot from 2020-03-12 02-31-03](https://user-images.githubusercontent.com/10780557/76507254-b3aa7f00-6409-11ea-8eeb-27a83a5f3246.png)

I think this closes this issue, but opens another. I'm not entirely sure what exploded yet, though, so I'm not 100% sure what to put in the new issue. It's quite possible that the stick was written correctly, but hit an exception cleaning up. It left the source and destination mounted:

```
tjcarter@chiyo:~$ pydf
Filesystem              Size  Used Avail  Use%               Mounted on                            
/dev/chiyo--vg/root     885G  602G  237G  68.1 [#######....] /                                     
/dev/nvme0n1p2          488M   82M  371M  16.7 [##.........] /boot                                 
/dev/nvme0n1p1          511M 5328k  506M   1.0 [...........] /boot/efi                             
/dev/loop0             5170M 5170M     0 100.0 [###########] /media/woeusb_source_1584008673_147536
/dev/sdc1                14G 5238M 9421M  35.7 [####.......] /media/woeusb_target_1584008673_147536
/dev/mapper/sdb1_crypt 3667G 1111G 2369G  30.3 [###........] /srv                                  
```

I can't reboot chiyo just now to test it booting UEFI. The other machine on hand is a Dell whose BIOS maker had their head inserted somewhere when they made the system UEFI-bootable from SATA, but not USB. Yes, really. *sigh* I think grub can work around that, but perhaps not in a way that'll make Windows install in UEFI mode.

**Comment by WaxyMocha** [comment]
Wait, what?  This just saves number to file, there is nothing that can blow up here with `OSError: Invalid argument`...
```python
with open("/proc/sys/vm/dirty_background_bytes", "w") as dirty_background_bytes:
        dirty_background_bytes.write(vm_dirty_background_bytes)
```
I will need investigate that...

**Comment by iKarith** [comment]
It did start in legacy mode. Popping it back into chiyo to have a look, I see:

```
tjcarter@chiyo:~$ sudo parted /dev/sdc p
Model: SanDisk Ultra (scsi)
Disk /dev/sdc: 15.4GB
Sector size (logical/physical): 512B/512B
Partition Table: msdos
Disk Flags: 

Number  Start   End     Size    Type     File system  Flags
 1      4194kB  15.4GB  15.4GB  primary  ntfs
 2      15.4GB  15.4GB  524kB   primary               lba
```

There doesn't appear to be a filesystem on sdc2. Is that where grub lives?

Looking at the source, it looks like the last thing it did would have been to install grub legacy boot (which worked). It didn't set the first partition bootable, but the Dell booted it without that.

**Comment by WaxyMocha** [comment]
I believe that it should be fat16, but I don't know why it is created.
```python
def create_uefi_ntfs_support_partition(target_device):
    """
    :param target_device:
    :return: None
    """
    utils.check_kill_signal()

    # FIXME: The partition type should be `fat12` but `fat12` isn't recognized by Parted...
    # NOTE: The --align is set to none because this partition is indeed misaligned, but ignored due to it's small size

    subprocess.run(["parted",
                    "--align", "none",
                    "--script",
                    target_device,
                    "mkpart",
                    "primary",
                    "fat16",
                    "--", "-1024s", "-1s"])
```
